### PR TITLE
Wip/hughsie/libcurl

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: src/fu-util.c [hunk 1]
```diff
@@ -674,6 +674,35 @@ fu_util_display_current_message (FuUtilPrivate *priv)
 	}
 }
 
+static gboolean
+fu_util_download(FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autofree gchar *basename = NULL;
+	g_autoptr(GBytes) blob = NULL;
+
+	/* handle both forms */
```
Comment (superm1) [inline src/fu-util.c:683]: comment seems wrong

File: src/fu-util.c [hunk 2]
```diff
@@ -3344,6 +3373,13 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: command description */
 		     _("Get all enabled plugins registered with the system"),
 		     fu_util_get_plugins);
+	fu_util_cmd_array_add(cmd_array,
+			      "download",
+			      /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+			      _("LOCATION"),
+			      /* TRANSLATORS: command description */
+			      _("Download a file using the client"),
```
Comment (superm1) [inline src/fu-util.c:3381]: I don't think "using the client" is really needed.  Isn't that a detail that doesn't need to be exported "how" it's downloaded?

File: src/fu-util.c (hunk 1 above)
Comment (hughsie) [reply src/fu-util.c]: Fixed, thanks.

File: src/fu-util.c (hunk 2 above)
Comment (hughsie) [reply src/fu-util.c:3381]: True, fixed.

**Review by superm1 — APPROVED** [review]

**Pull request merged**